utils/lib/Quickstart.ts

```diff
@@ -181,7 +186,7 @@ class Quickstart {
       id: id ? id : MOCK_UUID,
       dryRun,
       quickstartMetadata,
-    };
+    } as QuickstartMutationVariable;
```

Comment:
Typescript should determine this automatically
```suggestion
    };
```

Reply:
done